content/warnings/dont-call-proptypes.md

```diff
@@ -4,47 +4,48 @@ layout: single
 permalink: warnings/dont-call-proptypes.html
 ---
 
-> Note:
+> Uwaga:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> Z wersją React v15.5 `React.PropTypes` zostało przeniesione do innej paczki. Zamiast importować z paczki Reacta, proszę używaj [biblioteki `prop-types`](https://www.npmjs.com/package/prop-types).
```

Comment:
```suggestion
> Z wersją React v15.5 `React.PropTypes` zostało przeniesione do innej paczki. Zamiast importować z paczki Reacta, używaj [biblioteki `prop-types`](https://www.npmjs.com/package/prop-types).
```
Nigdzie indziej w tłumaczeniu nie prosiliśmy :-) Można spokojnie pominąć, bo zmienia wydźwięk. To jest dokumentacja, tu się nie prosi, tylko stwierdza fakty.

Reply:
no fakt, jakoś po angielsku jak się jest grzecznym to lepiej brzmi ;)